### PR TITLE
Update Submariner maintainers

### PR DESCRIPTION
**Pull request by dfarrell07**

### CODE REVIEW

File: project-maintainers.csv [hunk 1]
```diff
@@ -897,12 +897,17 @@ Sandbox,Fluid,Rong Gu,Nanjing University,RongGu,https://github.com/fluid-cloudna
...
+,,Janki Chhatbar,Red Hat,Jaanki,https://github.com/submariner-io/subctl/blob/devel/CODEOWNERS.in
+,,Maayan Friedman,Red Hat,maayanf24,
+,,Mike Kolesnik,Red Hat,mkolesnik,
 ,,Stephen Kitt,Red Hat,skitt,
+,,Sridhar Gaddam,Red Hat,sridhargaddam,
 ,,Thomas Pantelis,Red Hat,tpantelis,
-,,Mike Kolesnik,Red Hat,mkolesnik,
-,,Sridhar Gaddam,Red Hat,sridhargaddam ,
+,,Vishal Thapar,Red Hat,vthapar,
+,,Yosi Boaron,Red Hat,yboaron,
```
Comment (skitt) [inline project-maintainers.csv:910]:
```suggestion
,,Yossi Boaron,Red Hat,yboaron,

```

Comment (dfarrell07) [reply project-maintainers.csv]: Done, thanks

**Review by skitt — APPROVED** [review]

**Review by Jaanki — APPROVED** [review]

**Comment by Jaanki** [comment]
@Cmierly when you have a moment, could you please review and approve this PR if everything looks good? I’d appreciate any feedback!

**Pull request merged**